### ISSUE TITLE
Ground truth poses are not avaialble for sequence 11

### ISSUE THREAD

**Issue by Huleyun** [IIPCVLAB/LCCNet#19]
你好，
        我这边配好了环境，预测部分能够使用，但是在训练时报出这样的错误。

官网貌似只提供00--10的ground truth，但是直接运行train_with_sacred.py 会报这个错误。

> WARNING - LCCNet - No observers have been added to this run
> INFO - LCCNet - Running command 'main'
> INFO - LCCNet - Started
> Loss Function Choice: combined
> Val Sequence:  00
> Ground truth poses are not avaialble for sequence 11.
> Ground truth poses are not avaialble for sequence 12.
> Ground truth poses are not avaialble for sequence 13.
....
> Ground truth poses are not avaialble for sequence 21.
> VAL SET: Using this file: /home/usr/Pycharm_projects/data_odometry_color/dataset/sequences/val_RT_left_seq00_1.00_0.10.csv
> /home/usr/Pycharm_projects/LCCNet/venv/lib/python3.8/site-packages/torchvision/models/_utils.py:135: > UserWarning: Using 'weights' as positional parameter(s) is deprecated since 0.13 and may be removed in the future. Please use keyword parameter(s) instead.
  warnings.warn(
> /home/usr/Pycharm_projects/LCCNet/venv/lib/python3.8/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=ResNet18_Weights.IMAGENET1K_V1`. You can also use `weights=ResNet18_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
> Number of the train dataset: 39011
> Number of the val dataset: 4541
> 163
> 19
> Loading weights from ./pretrained/kitti_iter5.tar
> Number of model parameters: 69987887
> full training time = 0.00 HR
> INFO - LCCNet - Completed after 0:00:03`

**Comment by hungsing92** [comment]
你好，我也遇到这个问题，请问你解决了吗？可以给点建议不？

**Comment by hungsing92** [comment]
11之后的外参是有的，只是pose没有，把读取pose的代码注释掉。。。

**Comment by Hliu0313** [comment]
> 11之后的外参是有的，只是pose没有，把读取pose的代码注释掉。。。

[关于pose.txt/calib.txt真正含义](https://blog.csdn.net/yangziluomu/article/details/78339575) calib.txt文件中Tr才是外参，odom根本不重要
.

**Comment by str199044** [comment]
你好，请问这个问题应该怎么解决？

**Comment by Hliu0313** [comment]
| |
刘辉
|
|
***@***.***
|
直接点进去 pykitti中odom文件，注释读取odom那一段就行了。我们只是用Tr不使用odom
---- Replied Message ----
| From | Thore ***@***.***> |
| Date | 10/13/2023 15:59 |
| To | ***@***.***> |
| Cc | ***@***.***> ,
***@***.***> |
| Subject | Re: [IIPCVLAB/LCCNet] Ground truth poses are not avaialble for sequence 11 (Issue #19) |

你好，请问这个问题应该怎么解决？

**Comment by xuefeng1130** [comment]
> | | 刘辉 | | ***@***.*** | 直接点进去 pykitti中odom文件，注释读取odom那一段就行了。我们只是用Tr不使用odom ---- Replied Message ---- | From | Thore ***@***.***> | | Date | 10/13/2023 15:59 | | To | ***@***.***> | | Cc | ***@***.***> , ***@***.***> | | Subject | Re: [IIPCVLAB/LCCNet] Ground truth poses are not avaialble for sequence 11 (Issue #19) | 你好，请问这个问题应该怎么解决？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

直接点进去 pykitti中odom文件，注释读取odom那一段就行了。我们只是用Tr不使用odom
请问这个怎么做？

**Comment by xuefeng1130** [comment]
> 11之后的外参是有的，只是pose没有，把读取pose的代码注释掉。。。

我不知道怎么注释，希望得到您的帮助

**Comment by thompsoncjh** [comment]
找到pykitti包里面的对应py文件，注释掉报错的内容就行了

**Comment by xuefeng1130** [comment]
这是来自QQ邮箱的自动回复邮件。
您好，我的邮箱已接收到您发来的邮件。待我处理后给您回复。
